docs/openapi/orchestration.swagger.json

```diff
@@ -481,7 +481,23 @@
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/StakingServicePerformWorkflowStepBody"
```

Comment:
Not sure why this change showed up. Underlying package update is my best guess. I verified that they're functionally the same though.